### PR TITLE
Notebook and documentation fixes [skip ci]

### PR DESCRIPTION
**Pull request by nvkevlu**
Update menu of examples in documentation, remove irrelevant sections and section added back probably unintentionally in previous PR, some fixes in notebooks for language and grammar.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [x] Documentation updated.

### CODE REVIEW

File: docs/example_applications_algorithms.rst [hunk 1]
```diff
@@ -12,17 +12,24 @@ NVIDIA FLARE has several tutorials and examples to help you get started with fed
...
+   Federated Linear Model with Scikit-learn (GitHub) <https://github.com/NVIDIA/NVFlare/tree/main/examples/advanced/sklearn-linear>
+   Federated K-Means Clustering with Scikit-learn (GitHub) <https://github.com/NVIDIA/NVFlare/tree/main/examples/advanced/sklearn-kmeans>
+   Federated SVM with Scikit-learn (GitHub) <https://github.com/NVIDIA/NVFlare/tree/main/examples/advanced/sklearn-svm>
+   Federated Learning for Random Forest based on XGBoost (GitHub) <https://github.com/NVIDIA/NVFlare/tree/main/examples/advanced/random_forest>
    Hello MONAI Bundle (GitHub) <https://github.com/NVIDIA/NVFlare/tree/main/examples/hello-monai-bundle>
    Differential Privacy for BraTS18 Segmentation (GitHub) <https://github.com/NVIDIA/NVFlare/tree/main/examples/brats18>
    Prostate Segmentation from Multi-source Data (GitHub) <https://github.com/NVIDIA/NVFlare/tree/main/examples/prostate>
+   examples/federated_statistics
    Federated Policies (GitHub) <https://github.com/NVIDIA/NVFlare/blob/main/examples/federated-policies>
+   examples/tensorboard_streaming
```
Comment (YuanTingHsieh) [inline docs/example_applications_algorithms.rst:32]: I will go update the content inside that file.

File: docs/example_applications_algorithms.rst [hunk 2]
```diff
@@ -12,17 +12,24 @@ NVIDIA FLARE has several tutorials and examples to help you get started with fed
...
    examples/hello_tf2
-   examples/tensorboard_streaming
-   examples/federated_statistics
-   examples/xgboost
-   Hello Cyclic Weight Transfer (GitHub) <https://github.com/NVIDIA/NVFlare/blob/main/examples/hello-cyclic>
+   FL Simulator Notebook (GitHub) <https://github.com/NVIDIA/NVFlare/blob/main/examples/tutorials/flare_simulator.ipynb>
+   Hello FLARE API Notbook (GitHub) <https://github.com/NVIDIA/NVFlare/blob/main/examples/tutorials/flare_api.ipynb>
+   NVFLARE in POC Mode (GitHub) <https://github.com/NVIDIA/NVFlare/blob/main/examples/tutorials/setup_poc.ipynb>
    Federated Learning with CIFAR-10 (GitHub) <https://github.com/NVIDIA/NVFlare/tree/main/examples/cifar10>
+   examples/xgboost
```
Comment (YuanTingHsieh) [inline docs/example_applications_algorithms.rst:22]: I will go update the content inside that file.

File: examples/tutorials/flare_simulator.ipynb [hunk 3]
```diff
@@ -10,216 +10,7 @@
...
-    "\n",
-    "To introduce the FL Simulator, we'll run the `hello-numpy-cross-val` example located in the `NVFlare/examples/tutorial/hello-numpy-cross-val` directory."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "id": "e5ffe941",
-   "metadata": {},
-   "source": [
-    "Here we can see the basic configuration of a FLARE application in the `hello-numpy-cross-val/app` directory.  This includes a `config/` directory with client and server configurations in [`config_fed_client.json`](example/hello-numpy-cross-val/app/config/config_fed_client.json) and [`config_fed_server.json`](examples/hello-numpy-cross-val/app/config/config_fed_server.json).\n",
```
Comment (YuanTingHsieh) [inline examples/tutorials/flare_simulator.ipynb:31]: I am fine with this removal, tag @chesterxgchen for confirm

**Comment by nvkevlu** [comment]
Readme links to notebooks were added in a previous PR.

The left panel structure should be reflected by the TOC on the examples page.

**Pull request merged**